projects/components/src/datagrid/interfaces/datagrid-column.interface.ts

```diff
@@ -91,5 +92,18 @@ export interface GridColumn<R> {
      * TODO: Should this be made to work with top level search on grids across all columns?
      *  The above to-do is going to be worked on as part of https://jira.eng.vmware.com/browse/VDUCC-27 and
      */
-    filter?: ComponentRendererSpec<R, unknown>;
+    filterRendererSpec?: ComponentRendererSpec<R, unknown>;
+
+    /**
+     * Wired to ClrDgSortBy input to enable custom sorting feature offered by the Clarity data grid. This is also used
+     * to enable sorting independently from filtering when clrDgField is given as input
+     */
+    sortBy?: ClrDatagridComparatorInterface<R> | string;
```

Comment:
Because we only support sorting from the server side, what does it mean if the user passes in a `ClrDatagridComparatorInterface` to the column?